src/bin/repl/helper.rs

```diff
@@ -84,5 +87,27 @@ impl Hinter for ReplHelper {
 impl Completer for ReplHelper {
     type Candidate = String;
```

Comment:
Now that the commands are `&'static str`, this could be as well.
```suggestion
    type Candidate = &'static str;
```